comid-extensions.cddl

```diff
@@ -13,7 +13,7 @@ spdm-indirect-map = {
 ; Flags map extensions
 ;
 $$flags-map-extension //= (
-  ? &(not-runtime-meas: 6) => bool
-  ? &(not-immutable: 7) => bool
-  ? &(not-tcb: 8) => bool
+  ? &(runtime-meas: 6) => bool
+  ? &(immutable: 7) => bool
+  ? &(tcb: 8) => bool
```

Comment:
Will names like `runtime-measured`, `is-immutable`, and `is-tcb` be easier to understand?

Is `tcb` redundant? Since `replay-protected` and `integrity-protected` already implies that the attester is a TCB.

I think we need new flag `confidentiality-protected`.